### ISSUE TITLE
Plugin updated with skpm won't install

### ISSUE THREAD

**Issue by larsenwork** [skpm/skpm#162]
### Plugin in question:
https://github.com/larsenwork/sketch-easing-gradient 

### Steps
```
1. Successfully run `skpm publish patch` in the branch
2. Sketch picks up the update 
3. Click the `Update` button but nothing happens
```

### Screenshot
![image](https://user-images.githubusercontent.com/3859425/40617991-568bf906-6290-11e8-8837-309aab786ce5.png)

**Comment by mathieudutour** [comment]
It's a bug in Sketch, I filled an issue in our repo so I can track it there (BohemianCoding/Sketch#18523)

**Comment by mathieudutour** [comment]
Actually no it's not a bug in Sketch. You specify the version in your manifest.json so skpm doesn't use the one specified in the package.json. That means that your published version `0.4.2` is actually still `0.4.0` so Sketch cannot find the updated plugin.
